### PR TITLE
Added test for GetOrphanTasks

### PR DESCRIPTION
**Pull request by jakobht**
**What changed?**
Added tests

**Why?**
Increase code coverage

**How did you test it?**

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

**Comment by 3vilhamster** [comment]
I don't think this test adds anything to our code base. Please, focus on the functionality that is actually used. This looks like a test purely for coverage :(

File: common/persistence/nosql/nosql_task_store_test.go [hunk 1]
```diff
@@ -23,11 +23,14 @@
 package nosql
 
 import (
+	ctx "context"
```
Comment (taylanisikdemir) [inline common/persistence/nosql/nosql_task_store_test.go:26]: in general such aliasing of imports should be avoided to make the code across packages consistent for readability purposes. it should be done only for collision cases but we have a lot of bad examples in the code base.

File: common/persistence/nosql/nosql_task_store_test.go [hunk 2]
```diff
@@ -38,3 +41,25 @@ func TestNewNoSQLStore(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, store)
 }
+
+func setupNoSQLStoreMocks(t *testing.T) *nosqlTaskStore {
```
Comment (taylanisikdemir) [inline common/persistence/nosql/nosql_task_store_test.go:45]: I guess you need to rebase

**Review by taylanisikdemir — APPROVED** [review]

File: common/persistence/nosql/nosql_task_store_test.go (hunk 1 above)
Comment (jakobht) [reply common/persistence/nosql/nosql_task_store_test.go]: I don't remember doing this, but changed it to just context

File: common/persistence/nosql/nosql_task_store_test.go (hunk 2 above)
Comment (jakobht) [reply common/persistence/nosql/nosql_task_store_test.go]: Rebased

**Pull request merged**